ci/maven.yml

```diff
@@ -21,4 +23,4 @@ jobs:
       with:
         java-version: 1.8
     - name: Build with Maven
-      run: mvn -B package --file pom.xml
+      run: mvn --batch-mode --update-snapshots verify
```

Comment:
i branched off the old default branch :(

Reply:
Serkan